iohk-monitoring/src/Cardano/BM/Output/EKGView.lhs

```diff
@@ -177,7 +177,7 @@ instance ToObject a => IsBackend EKGView a where
         ekghdl <- getLabel "iohk-monitoring version" ehdl
         Label.set ekghdl $ pack (showVersion version)
         ekgtrace <- ekgTrace ekgview config
-        queue <- atomically $ TBQ.newTBQueue 512
+        queue <- atomically $ TBQ.newTBQueue 25120
```

Comment:
Where this number comes from? I mean `25120`.

Reply:
just larger.